lldb/tools/lldb-dap/DAP.h

```diff
@@ -209,6 +230,7 @@ struct DAP {
   // empty; if the previous expression was a variable expression, this string
   // will contain that expression.
   std::string last_nonempty_var_expression;
+  Gotos goto_id_map;
```

Comment:
This is more than just a map, right?
```suggestion
  Gotos gotos;
```